src/app/components/seller/seller-product/seller-product.component.ts

```diff
@@ -17,22 +17,22 @@ export class SellerProductComponent implements OnInit {
   @ViewChild('viewMore') viewMore: ViewMoreComponent;
   @ViewChild('closeEdit') closeEdit: ElementRef<HTMLElement>;
   cproduct: CreateProductResponse[];
-  
-  constructor( private productservice: ProductsService) {}
-  ngOnInit(): void {}
 
- 
-  cp(createproduct: CreateProductResponse):void{
+  constructor(private productservice: ProductsService) { }
+  ngOnInit(): void { }
+
+
+  cp(createproduct: CreateProductResponse): void {
```

Comment:
Rename this function to a meaningful name

Reply:
this function was created by Daud 3 months ag, my editor formated i